builder/vsphere/supervisor/step_import_image.go

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+//go:generate packer-sdc struct-markdown
+//go:generate packer-sdc mapstructure-to-hcl2 -type ImportImageConfig
+
+package supervisor
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+	"github.com/vmware-tanzu/image-registry-operator-api/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ImportTargetKind       = "ContentLibrary"
+	ImportTargetAPIVersion = "imageregistry.vmware.com/v1alpha1"
+
+	DefaultWatchImageImportTimeoutSec = 600
+
+	StateKeyImageImportRequestCreated = "item_import_req_created"
+)
+
+var IsWatchingImageImport bool
+
+type ImportImageConfig struct {
+	// The name of the image import request. If not specified, the name defaults to `packer-vsphere-supervisor-import-req-<random-suffix>`.
```

Comment:
```suggestion
	// The name of the image import request.
	// Defaults to `packer-vsphere-supervisor-import-req-<random-suffix>`.
```

Reply:
Thanks, will update to this format for all configs.